src/resources/benefitCompany.ts

```diff
@@ -29,17 +29,17 @@ export const BenefitCompanyResource: ResourceIF = {
         helpText: [
           `The sample Incorporation Agreement and Benefit Company Articles is a template that you can use
             to create an incorporation agreement and articles for your company. It uses all the standard
-            provisions suggested by legislation and also includes a place to specify the company’s benefit
```

Comment:
This apostrophe is fine in a word processor but its use is inconsistent here, so I changed it to the usual apostrophe (in a few places).